app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+
+
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+
+    if command_parts[1] == command_parts[2]:
+        return
+
+    if command_parts[0] != "cp":
+        print("Wrong command use 'cp'.")
+        return
+
+    if not os.path.exists(command_parts[1]):
+        print(f"File {command_parts[1]} not exist.")
+        return
+
+    try:
+        with (open(command_parts[1], "r") as source_file,
+              open(command_parts[2], "w") as destination_file):
```

Comment:
Clean Code: There are no comments in your code. It's a good practice to add comments to your code to make it easier for others to understand what it does. For example, you could add a comment before this block explaining that it opens the source file and the destination file and copies the content from the source file to the destination file.